### PR TITLE
Extend term-aggregation options in api #4407

### PR DESCRIPTION
**Pull request by anatol-sialitski**

### CODE REVIEW

File: modules/core/core-api/src/main/java/com/enonic/xp/query/aggregation/TermsAggregationQuery.java [hunk 1]
```diff
@@ -50,7 +50,7 @@ public Type getOrderType()
         return orderType;
     }
 
-    public long getMinDocCount()
+    public Long getMinDocCount()
```
Comment (rymsha) [inline modules/core/core-api/src/main/java/com/enonic/xp/query/aggregation/TermsAggregationQuery.java:53]: This is impossible to change return type in API.

File: modules/core/core-api/src/main/java/com/enonic/xp/query/aggregation/TermsAggregationQuery.java [hunk 2]
```diff
@@ -103,7 +103,7 @@ public Builder size( final Integer size )
             return this;
         }
 
-        public Builder minDoccount( final long minDocCount )
+        public Builder minDocCount( final Long minDocCount )
```
Comment (rymsha) [inline modules/core/core-api/src/main/java/com/enonic/xp/query/aggregation/TermsAggregationQuery.java:106]: it is impossible to change method name and arg types in API

**Review by rymsha — CHANGES_REQUESTED** [review]

File: modules/core/core-api/src/main/java/com/enonic/xp/query/aggregation/TermsAggregationQuery.java (hunk 1 above)
Comment (anatol-sialitski) [reply modules/core/core-api/src/main/java/com/enonic/xp/query/aggregation/TermsAggregationQuery.java]: Done

File: modules/core/core-api/src/main/java/com/enonic/xp/query/aggregation/TermsAggregationQuery.java (hunk 2 above)
Comment (anatol-sialitski) [reply modules/core/core-api/src/main/java/com/enonic/xp/query/aggregation/TermsAggregationQuery.java]: Done

**Review by vbradnitski — APPROVED** [review]

**Pull request merged**